### DISCUSSION TITLE
"Rear-Admiral" replacement "Drill Sergant" is not adequate

### DISCUSSION THREAD

**Discussion by TaipionsSalemClient** [PavelZinchenko/event-horizon-frontier-builds#74] [Ideas]
Hands down, extra xp is absolutely useless, as mercs will:

get a ton less exp as you anyways
deal a ton less dmg than you anyways
Furthermore, the new perk is only (slightly) useful early on,
whereas read-admiral was meant for endgame (though useful in any game stage)

So you are basically replacing an endgame perk with a super weak (by comparison) early game perk!

=> forces literally everyone to literally dump and replace literally every single merc pilot,
this can't be serious

solution:
replace rear admiral with something else instead, that is:

endgame oriented
comparable to the remaining useful skills in desirability (Armourer, Cannoneer, Technician)
(suggestions:)
=> mixed perk with +def and +dmg, 15% each, stacks with Armourer and Cannoneer
or something else that is roughly the same power for instance: slight bonus to max energy + energy regen, or resists of sorts, or fire rate, or anything really... as long as it's on a comparable lvl)

**Comment by PavelZinchenko** [comment]
Please suggest alternative perks that could be a decent replacement for Rear Admiral. I'll choose the one I like the most

**Comment by IsmollCheeKen** [comment]
To a provide a baseline for future ideas, Drill Sergeant right now is replaceable with the Cannoneer/Armourer skills, these two skills are the next best skills on the line. So, the Drill Sergeant replacement skill should be the same level as Cannoneer/Armourer or higher, but not too overpowered. I'll get back on this discussion once I have an idea.

**Comment by imverybadatnames** [comment]
i want to post an idea, but im trash at making ideas :((((

welp here we go
Perk idea 1.
"Veteran" perk.
Icon: A veteran icon just like the icon of Veteran ships in the original EH.

Veteran perk effect: I have 3 ideas for the perk of the veteran.
Idea 1: A small increase (like 15%) in both ship attack and defense.
Idea 2: After reaching a certain condition (such as piloting the same ship for extended periods of time, or being a mercenery for long enough), will increase ship attack and defense by a decent amount (20%-35%).
Idea 3: a minor boost in every stat (small increase in ship stats, slightly faster respawn, etc).

Perk idea 2.
"Electrician" Perk.
Icon: electricity (like the fire button image of the Heavy lightning cannon.)
Electrician perk effect: I have 2 ideas for the electrician perk ability.
Idea 1: a boost in the ship's energy capacity and regeneration (around 15-30%)
Idea 2: Weapons fired will do some sort of EMP like the effect of the HLC (effect could be limited to Energy-type weapons only)

Anyways, there are my 2 (terrible) ideas for the replacement of Drill Sergeant.

**Reply by IsmollCheeKen** [reply]
I like that Veteran perk, +15% Attack and +15% Defense, it's a good alternative to Cannoneer/Armourer, with equal trade-offs

**Comment by Inedilarion** [comment]
1) Увеличение дальнобойности оружия/дронов и радиуса его поражения (возможно, как дополнение к бусту урона и защиты)
2) Добавить действие какого-либо девайса (возможно отредактированное). Например, маскировка от ракет/дронов, телепорт корабля при угрозе попадания по нему вражеского снаряда, короткая неуязвимость при смерти (без восполнения хп и энергии), дальнобойная не тратящая энергию и редко атакующая система точечной защиты и т.п.
3) Повышенный приоритет корабля для врагов, чтобы они с меньшей вероятностью пытались атаковать базу
4) Иммунитет ко всему, кроме урона. То есть, к отбрасыванию, к замедлению, к системе захвата дронов (клонам и дронам), к раскручиванию, к ЭМИ, к зацеплению крюков

1. Increasing the range of weapons/drones and their damage radius (possibly in addition to a boost in damage and defense)
2. Add an action of a device (possibly edited). For example, camouflage from missiles/drones, teleport of a ship when there is a threat of an enemy shell hitting it, short-term invulnerability upon death (without replenishing HP and energy), long-range point defense system that does not waste energy and rarely attacks, etc.
3. Increased ship priority for enemies so they are less likely to try to attack the base
4. Immunity to everything except damage. That is, to the knockback, to the slowdown, to the drone capture system (clones and drones), to the spin-up, to the EMP, to the engagement of hooks

**Reply by IsmollCheeKen** [reply]
I see only No.4 actually being usable as it is general purpose.

No.1 is very specialized for drone carriers - which the majority of players' mercenary pilots are not meant for. Even so, it would be quite pointless to have a Drone Bonus alongside a Specialist pilot for literally any ship that is not meant to be a drone carrier.

No.2 is quite underwhelming, while it is a miscellaneous bonus, you'd still prefer bonuses that increases stats, which would either be Cannoneer/Armourer.

No.3 is similar to a Taunt, and I believe it would cause a lot of AI behavior problems, even if only one ship has it, let alone all ships on the field.

**Reply by Inedilarion** [reply]
About 1 you are wrong, the range and radius for weapons will increase the effectiveness of any ship.
2 can be made very strong, for example Invader's infinity stone

**Reply by IsmollCheeKen** [reply]
No.1 +Range and +AoE is outclassed by Cannoneer/Armourer (C/A), if we're going to look at how the new Merc bonuses are calculated (with Specialist included) - pilots without C/A will only have a boost of x4.950, as opposed to the boost of pilots with C/A which is x5.940. That right there is a huge difference for you to be trading it for +Range and +AoE (which can be attained from modules and gold modifiers anyway) . Also, the +AoE is semi-useful as not all weapons have AoE.

For No.2, same reason as No.1 above, you could get Armourer and have a defense boost of x5.940 instead of short-term invulnerability where you would probably just die after it wears off. The only ship classes which I see the short-term invulnerability is useful on is for ships that are Cruiser class and below, since Armourer on them practically does not make a difference.

**Reply by Inedilarion** [reply]
You don't understand how boosts work. 5940 instead of 4950 is 5940/4950 = 1.2 boost, that is, +20% damage/hp.
Now let's move on to n. 1.
![Screenshot_2024-03-17-23-49-17-239_com ZipasGames Frontier](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/66635560/287d3ea6-699a-4c76-8247-507a2ec41a25)
Oracle with a HLC at +50 range. If the captain's boost gives +50% range, it will be possible to replace the cannon with +200% damage, doubling the DPS and improving energy efficiency. If the bonus is +30%, a 160% bonus from rangemasters would be needed to maintain the existing range, meaning it would be possible to replace 4 rangemasters with reactors and focusers, which would increase damage by over 20% (even while saving 80% of the extra energy costs from rangemasters in focusers, the final damage will become from 380% to 460%, and 460/380=1.21).
![Screenshot_2024-03-17-22-28-47-295_com ZipasGames Frontier](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/66635560/dff69426-8803-4711-8faa-745682fe4072)
Similar calculations can be made with a Scarab, on which, in order to maintain range, it will be necessary to remove 5-6 rangemasters, installing an extra reactor and focusers. As for ships without rangemasters, they will be able to start shelling the enemy from a greater distance, which will reduce the time from enemy spawn to destruction in most cases. Also in n. 1 there was a note that the bonus to range can be combined with the bonus to damage/hp if it is not enough
Regarding n. 2, didn’t you yourself write here:
[https://github.com/PavelZinchenko/event-horizon-frontier-builds/issues/112](url)
that such a bonus for the Invader is too strong? In my list I tried to somehow eliminate this problem, or at least give an idea of how this can be done

**Reply by IsmollCheeKen** [reply]
Alright, +Range is sound in No.1, with the right +X% boost, it can be on par with C/A.

For No.2, I was vouching for the Space Invader to die reasonably, instead of being outright immortal or reviving itself 20 times before fully dying, that is unreasonable. Your suggestion of temporary invulnerability before death is sound but I fail to see how that is better than Armourer, especially for bigger ships (Battleships/Capitals). And it's not as if you can just tweak the invulnerability timer to be longer as well just to make it viable for bigger ships. Longer invulnerability time will boost Cruiser classes and below immensely coupled with Infinity Stone and their fast respawn rate.

**Comment by IsmollCheeKen** [comment]
An idea pitched in by ChaosFred;

Return the pilot perk back to Rear Admiral - Extra Command Points => More ships in Fleet => Allows for more Fleet customization => Allows players to handle more difficult content. Make Rear Admiral available on Battleship and Capital Ship pilots ONLY.

Also, he requests to increase maximum ship level cap to 200, just like in EH:RPG, Increased level caps => Stronger ships => Can clear more content => More stars allowance

**Comment by TaipionsSalemClient** [comment]
> An idea pitched in by ChaosFred;
> 
> Return the pilot perk back to Rear Admiral - Extra Command Points => More ships in Fleet => Allows for more Fleet customization => Allows players to handle more difficult content. Make Rear Admiral available on Battleship and Capital Ship pilots ONLY.
> 
> Also, he requests to increase maximum ship level cap to 200, just like in EH:RPG, Increased level caps => Stronger ships => Can clear more content => More stars allowance

The whole idea and discussion is about **removing** the command point perk, not re-adding it,
this "idea" is worse than "just revert rear admiral to what it was"! :D :D :D

Max ship lvl cap to 200?!?! why??!?! That's just the wrong kind of powercreep and would come with a ton of other issues like a ton of useless skill points flood and what not...

My opinion on this:
- bonus to range will be highly OP and too situational, not all ships benefit from it, some will benefit too much, Rear Admiral was a universally useful perk and so the replacement should also be that
=> +50% range as perk?! as a multiplier?! that's ludicrous! next thing you see is HLC turret ships sniping the enemy SB in arena...
- bonus to AoE range is just the same, not universally useful, not an option
-  same for energy and other bonuses... not universally useful, not a viable replacement
### looking at all this, the only viable replacement to rear admiral is some variation of +def and +dmg combined bonus like specialist but lower
- other perk ideas can be very well added on their own, but not as direct replacement for rear admiral, **but if you do, please give us a better way to find pilots!** it's already a pain and it will get worse with more perk!
=> idea: 1: double the pilots available in the "random list" on the mercenary npc, 2: give a new interface where you can simply select what perks (and all other details, name, race, gender,...) you want on a pilot, but the pilot costs 2x or 3x as much if bought there

**Comment by IsmollCheeKen** [comment]
I think everyone agrees on Veteran now being the best candidate, because right now, people would want either Cannoneer/Armourer, so Veteran with +15% on both Attack and Defense  would be great since it's on equal grounds as Cannoneer and Armourer.
